app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less

```diff
@@ -156,6 +156,43 @@
             } 
         }
     }
+    .product-options {
```

Comment:
Please, place product related styles to the Catalog module instead of Backend module